modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -434,7 +434,7 @@ spec:
 EOF
 ----
 +
-[Note] 
+[Note]
```

Comment:
Kind of out of scope for this PR, but minor thing:
```suggestion
[NOTE]
```

Reply:
I probably won't change it for this PR, but I'll try to catch it the next time I'm doing some random cleanup!